build-config.json

```diff
@@ -6,10 +6,11 @@
     "srcResources": "./src/content/**/*.@(jpg|jpeg|png|gif)",
     "srcDiagrams": "./src/content/**/*.@(dot|neato|fdp|sfdp|twopi|circo)",
     "srcContentBase": "./src/content",
-    "srcAssetImages": "./src/@(assets)/**/*.@(jpg|jpeg|png|gif)",
+    "srcStaticAssets": "./src/@(assets)/**/*.@(jpg|jpeg|png|gif|ttf)",
     "srcStyleEntry": "./src/assets/style.scss",
     "srcStylesAny": "./src/assets/**/*.scss",
     "vendorAssets": ["./node_modules/highlight.js/styles/atom-one-dark.css"],
     "invaderTagDefsBase": "./lib/invader/src/tag/hek/definition/"
-  }
+  },
+  "baseUrl": "https://c20.reclaimers.net"
```

Comment:
The site now needs to know its own base URL to build absolute URLs for opengraph meta tags.